### ISSUE TITLE
Time series data split

### ISSUE THREAD

**Issue by CSNoyes** [automl/auto-sklearn#300]
Could documentation be added on how to properly use auto-sklearn with time series data? It doesn't appear to be possible to pass in a TimeSeriesSplit or other cv type, and the documentation is unclear on whether or not samples are shuffled (thus introducing lookahead bias).

**Comment by mfeurer** [comment]
Thanks for pointing out that we need to document the behavior of Auto-sklearn. Auto-sklearn uses a StratifiedCrossValidation and shuffles the data. Therefore, it's not possible to use it for time series data at the moment.

**Comment by mfeurer** [comment]
This should now work, see #501.
